Code/D/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# How to Become a Contributor and Submit Your Own Code
+
+## Reporting Bugs
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes into a series of logical self-contained commits.
+1. For Neptune-versioned libraries, make sure you also follow the
+   [Neptune Library Contributions Guide](https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst).
+1. Submit your pull request against the appropriate branch.
+
+## Reviewing and Rebasing and Merging Patches
+
+1. All feedback should be professionally given and non-personal. Offensive,
+   intimidating, insulting, racist, homophobic, and sexist comments and
+   behavior will not be tolerated.
+1. Reviewers will comment on your code providing feedback to point out any
+   problems and improve the quality of the code. Feel free to respond to these
+   comments if you have any questions to clarify the meaning of the comments.
+1. Please try and pay attention to all the comments and address them all.
+1. Reviewers can mark a PR as "changes requested" or "approved".
+1. Once a PR has been approved by at least one reviewer and they are no
+   outstanding comments that have not been addressed from other reviewers,
+   either a reviewer or the submitter (providing they have push access) can
+   rebase and merge the PR.
+1. Repo owners try to review PRs promptly, you can feel free to give a "Ping!"
```

Comment:
"Repo owners will try to review PRs promptly, but feel free to ..." is a bit clearer.